docs/user/dashboard/lens.asciidoc

```diff
@@ -220,3 +227,43 @@ Visualizing saved searches in unsupported in *Lens*.
 ===== How do I change the number of suggestions?
 
 Configuring the *Suggestions* that *Lens* automatically populates is unsupported.
+
+[float]
+[[is-it-possible-to-use-different-indexpatterns-in-lens]]
+===== Is it possible to use different index patterns in a single visualization?
+
+You can create a visualization from multiple index patterns in *Lens*: this feature is available for a subset of visualizations such *Bar*, *Line* and *Area* charts.
+
+Each *Layer* in a visualization is associated with an index pattern and mutiple *Layers* can be combined together within the same visualization. Each *Layer* also has a chart switcher button in order to select the best type of visualization for the specific dataset.
+At the same time it is possible to dynamically change the indexpattern for the *Layer* when required.
+
+[float]
+[[why-my-field-x-is-missing-from-the-fields-list]]
+===== Why is my field X missing from the fields list in Lens?
+
+*Lens* does not support the visualization of full-text fields, therefore it is not showing them in the data summary.
+You will still see the `X.keyword` field, when available.
+
+[float]
+[[how-to-handle-gaps-in-time-series-visualizations]]
+===== How to handle gaps in time series visualizations?
+
+*Lens* provides a set of features to handle missing values for *Area* and *Line* visualization.
+This is useful for sparse data in time series.
+
+To select a different way to represent missing values, click on the *brush* button above the visualization in the editor to see a selection of strategies to handle missing values. The default strategy is to hide the gaps.
+
+[float]
+[[is-it-possible-to-change-the-scale-of-Y-axis]]
+===== Is it possible to change the scale of the Y axis in a visualization?
```

Comment:
```suggestion
===== Is it possible to statically define the scale of the Y axis in a visualization?
```
Nit: I think `change` can be misleading, but I want to hear more opinions about it!

Reply:
Btw Joe might be faster 😁 https://github.com/elastic/kibana/issues/67614